crates/sdl-encoder/src/field_type.rs

```diff
@@ -0,0 +1,65 @@
+use std::fmt::{self, Display};
+
+/// Define Field Type.
+#[derive(Debug, PartialEq, Clone)]
+pub enum FieldType {
```

Comment:
You may want to call this `Type` instead of `FieldType` because the same structure is used for types of arguments.

(In `graphql-js`, this would correspond to `GraphQLType`, and there is a whole hierarchy of types and type comparison functions that we may want to support if we extend this beyond the current introspection use case.)

Reply:
Hmmm yea that's a good point, thanks!